registration/include/pcl/registration/impl/correspondence_rejection_sample_consensus.hpp

```diff
@@ -133,9 +132,8 @@ CorrespondenceRejectorSampleConsensus<PointT>::getRemainingCorrespondences(
         inlier_indices_.push_back(index_to_correspondence[inlier]);
     }
 
-    // get best transformation
-    Eigen::VectorXf model_coefficients;
-    sac.getModelCoefficients(model_coefficients);
+    // get the best transformation
+    Eigen::VectorXf model_coefficients = sac.getModelCoefficients();
```

Comment:
```suggestion
    const Eigen::VectorXf& model_coefficients = sac.getModelCoefficients();
```
Similarly, could be a const reference.